packages/suite/src/actions/wallet/__tests__/discoveryActions.test.ts

```diff
@@ -631,18 +633,21 @@ describe('Discovery Actions', () => {
             useEmptyPassphrase: false, // mandatory
         });
         const store = initStore(state);
-        const fn = discoveryActions.getDiscoveryAuthConfirmationStatus;
+
+        // Necessary workaround, as redux-mock-store, as we use it, doesn't have immutable state.
+        // TODO: We really should mock redux in unit tests differently.
+        const fn = (state: any) => selectDiscoveryAuthConfirmation({ ...state });
```

Comment:
I think I already solved this issue, you can try use `configureMockStore` from `@suite-common/test-utils` it has some reasonable like default thunk middleware, reducers that you pass actually works (no need to do hacks with `store.subsribe(.....` and it should have nearly same api as redux-mock-store so it should be nearly drop-in replacement

Reply:
Also extra dependencies for thunks are automatically mocked, which is also nice.

Reply:
Apparently it's not drop-in replacement, at least because of some typing problems, and I don't want to postpone this PR even more, sorry. We'll fix it later.